packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
+    <a href="#" class="turn-off" role="button" @click="$emit('dndOff')">Turn Off</a>
+  </div>
+</template>
+
+<style scoped>
+.dnd-message-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 32px;
+  background-color: #7e48ff;
+  text-align: center;
+}
+
+.do-not-disturb {
+  font-style: oblique;
+}
+
+a.turn-off {
+  margin-left: 8px;
+  color: #fff;
```

Comment:
Same as before, you can use the color palette:
```suggestion
  color: var(--color-salt-10);
```